Bruin Bite/Network/UserManager.swift

```diff
@@ -172,6 +172,19 @@ class UserManager {
         }
     }
 
+    func invalidateAccessToken(token: String) {
+        self.provider.request(.logoutUser(token: token, client_id: CLIENTID, client_secret: CLIENTSECRET)) { result in
+            switch result {
+            case .success:
+                self.logOutUser()
```

Comment:
Add the two lines from the old logOutUser function here after changing this functions name

Reply:
Okie